content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-rego-policies.md

```diff
@@ -5,7 +5,7 @@ description: "Writing Rego-based policies for Chainguard Enforce"
 date: 2023-01-12T15:56:52-07:00
 lastmod: 2023-01-12T15:56:52-07:00
 draft: false
-tags: ["SBOM"]
```

Comment:
The ones I tagged with SBOM do have SBOM policies. I don't know how granular you would like to be

Reply:
Ah, I was thinking tagging at a higher level. But it makes sense to tag these with SBOMs. I'll add them back in,